src/Symfony/Bundle/TwigBundle/CacheWarmer/TemplateCacheCacheWarmer.php

```diff
@@ -75,4 +86,29 @@ public function isOptional()
     {
         return true;
     }
+
+    /**
+     * Find templates in the given directory.
+     *
+     * @param string $namespace The namespace for these templates
+     * @param string $dir       The folder where to look for templates
+     *
+     * @return array An array of templates of type TemplateReferenceInterface
+     */
+    private function findTemplatesInFolder($namespace, $dir)
+    {
+        $templates = array();
+
+        if (is_dir($dir)) {
+            $finder = new Finder();
+            foreach ($finder->files()->followLinks()->in($dir) as $file) {
+                $templates[] = new TemplateReference(
+                    sprintf('@%s/%s', $namespace, $file->getRelativePathname()),
```

Comment:
You can have a `null` or empty namespace?

So for config that looks like?

```yaml
twig:
    paths:
        "/path/to/templates": `
```

Should it be like this?

```php
$namespace ? sprintf('@%s/%s', $namespace, $file->getRelativePathname()) : $file->getRelativePathname()
```